deck/packages/core/src/manifest/stage/JobStageExecutionLogs.tsx

```diff
@@ -52,7 +52,7 @@ export class JobStageExecutionLogs extends React.Component<IJobStageExecutionLog
     const { manifest } = this.state;
     const { externalLink, podNamesProviders, location, account } = this.props;
     // prefer links to external logging platforms
-    if (externalLink) {
+    if (externalLink && (!externalLink.includes('{{') || !isEmpty(manifest))) {
```

Comment:
Think we need a rewrite or rethink on this... too complicated of logic.  I could see passing all of this down a bit and being a bit smarter on the checks internally.